grails-app/assets/javascripts/managementUnit.js

```diff
@@ -460,3 +463,31 @@ var ManagementUnitPageViewModel = function(props, options) {
         initialiseTabs(tabs, {tabSelector:'#managementUnit-tabs.nav a', tabStorageKey:'selected-managementUnit-tab'});
     };
 };
+
+function toggleStarred(isManagementUnitStarredByUser, userId, managementUnitId) {
```

Comment:
I understand this will have been copied from the project page, but the favourite feature was written before we introduced knockout.   This method should be a function in the ManagementUnitViewModel.  Please pass the selectors and baseUrl to the config so they aren't hard coded when moving it.

Reply:
This has now been committed.